code/_onclick/click.dm

```diff
@@ -115,7 +115,7 @@
 
 	if(HAS_TRAIT(src, TRAIT_HANDS_BLOCKED))
 		changeNext_move(CLICK_CD_HANDCUFFED)   //Doing shit in cuffs shall be vey slow
-		UnarmedAttack(A, FALSE, modifiers)
+		UnarmedAttack(A, Adjacent(A), modifiers)
```

Comment:
Clicking on anything while handcuffed automatically assumed you were not adjacent. 
I changed it to make monkey biting work properly. 
Some unarmed attack overrides may have relied on this weird facet, I went through and double checked to make sure but you never know.